gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/AuthConfig.groovy

```diff
@@ -16,19 +16,36 @@
 
 package com.netflix.spinnaker.gate.security
 
+import com.netflix.spinnaker.gate.security.rolesprovider.UserRolesProvider
 import groovy.util.logging.Slf4j
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
 import org.springframework.http.HttpMethod
 import org.springframework.security.config.annotation.web.builders.HttpSecurity
 
 @Slf4j
+@Configuration
 class AuthConfig {
 
+
+  @Bean
+  @ConditionalOnMissingBean(UserRolesProvider)
+  UserRolesProvider defaultUserRolesProvider() {
+    return new UserRolesProvider() {
+      @Override
+      Collection<String> loadRoles(String userEmail) {
+        return []
+      }
+    }
+  }
+
   static void configure(HttpSecurity http) throws Exception {
     http.csrf().disable()
     http.authorizeRequests()
-        .antMatchers(HttpMethod.OPTIONS, "/**").permitAll()
-        .antMatchers('/auth/user').permitAll()
-        .antMatchers('/health').permitAll()
-        .antMatchers('/**').authenticated()
+      .antMatchers(HttpMethod.OPTIONS, "/**").permitAll()
```

Comment:
Unless you feel strongly about these whitespace changes, you should remove them as changes in PRs prior to sending them out. They add clutter and we're just going to go back and forth adding/removing them.